helm/tweek/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for tweek.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+apiReplicaCount: 1
+gatewayReplicaCount: 1
+domain: gateway.192.168.99.100.xip.io
+
+imageVersions:
+  api: 1.0.0-rc1
+  authoring: 1.0.0-rc2
+  editor: 1.0.0-rc5
+  gateway: 1.0.0-rc5
+  git: latest
+  publishing: 1.0.0-rc2
+
+dbvolume:
+  size: 8Gi
+  storageClass: standard
+
+# set to url of your git repository to prevent creation of git pod.
+gitRepo: ""
+
+gitvolume:
+  size: 2Gi
+  storageClass: standard
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  tls: []
+  #  - secretName: chart-example-tls
+
+resources: {}
```

Comment:
the resources are not used, and should probably have sane defaults for each one of the services.

Reply:
Hmmm... If we define different resources for different services, the resources section can become pretty large.

Reply:
It just defaults, you don’t have to override them